metadeploy/api/serializers.py

```diff
@@ -602,3 +616,21 @@ class Meta:
             "company_logo",
             "favicon",
         )
+
+
+class ScratchOrgSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ScratchOrg
+        fields = (
+            "id",
+            "plan",
+            "email",
+            "enqueued_at",
+            "created_at",
+            "edited_at",
+            "status",
+            "org_id",
+        )
+
+    id = serializers.CharField(read_only=True)
+    plan = IdOnlyField(model=Plan)
```

Comment:
I think there are additional fields here which should be read-only. It should not be possible to override enqueued_at, created_at, edited_at, status, and org_id in the POST to request creation of a scratch org.